tests/function_single_destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -330,6 +330,8 @@ type T = ({
   items: number
 }) => void;
 
-const X = (props: { a: boolean }) => <A />;
+const X = (props: {
+  a: boolean
```

Comment:
This is interesting evidence that using visible groups is a cleaner approach here:

The "respect newlines in `ObjectTypeAnnotation`s" heuristic was being violated in the hugged-param case because there was no `group()` to force-break